docs/plugins/filter_processor.md

```diff
@@ -2,10 +2,14 @@
 
 This is used to filter transactions to include only the ones that you want. This may be useful for some deployments
 which only require specific applications or accounts. Using the default configuration, results for a matched transaction
-would also include its transaction group and search is disable for inner transactions. 
+would also include its transaction group and search is disabled for inner transactions.
+
+`TODO: How to apply the following sections?`
```

Comment:
^^^

Reply:
We have a document for the filter processor now, and it seems like there's a lot of overlap: https://github.com/algorand/conduit/blob/master/docs/tutorials/FilterDeepDive.md